mlrun/model_monitoring/db/tsdb/tdengine/tdengine_connector.py

```diff
@@ -220,23 +220,24 @@ def delete_tsdb_resources(self):
             "Deleting all project resources using the TDEngine connector",
             project=self.project,
         )
+        drop_statements = []
         for table in self.tables:
-            get_subtable_names_query = self.tables[table]._get_subtables_query(
-                values={mm_schemas.EventFieldType.PROJECT: self.project}
-            )
-            subtables = self.connection.run(
-                query=get_subtable_names_query,
-                timeout=self._timeout,
-                retries=self._retries,
-            ).data
-            drop_statements = []
-            for subtable in subtables:
-                drop_statements.append(
-                    self.tables[table]._drop_subtable_query(subtable=subtable[0])
+            drop_statements.append(self.tables[table]._drop_supertable_query())
+
+            try:
+                self.connection.run(
+                    statements=drop_statements,
```

Comment:
Not sure I understand the logic here.
Why do you try deleting the same table over and over in this for loop?
I think `drop_statements` shouldn't aggregate the super table names.